src/main/java/com/acrolinx/sidebar/live/LiveSuggestionFromJSON.java

```diff
@@ -0,0 +1,19 @@
+/**
+ *  Copyright (c) 2022-present Acrolinx GmbH
+ */
+package com.acrolinx.sidebar.live;
+
+@SuppressWarnings({"MismatchedReadAndWriteOfArray", "unused"})
+public class LiveSuggestionFromJSON {
+    private String preferredPhrase;
+    private String description;
+
+    public LiveSuggestionFromJSON() {
+        //
```

Comment:
reason for empty function?

You can throw UnsupportedOperationException it has to be empty

Reply:
Thought it would be needed for GSON, did not test any of that yet because it would have been used for eclipse, but I do not think these classes are necessary anymore. So I removed them 👍🏻